grammars/prql-lezer/test/misc.txt

```diff
@@ -124,7 +124,7 @@ group customer_id (
 
 ==>
 
-Query(Statements(PipelineStatement(Pipeline(CallExpression(Identifier,Identifier,NestedPipeline(Pipeline(CallExpression(Identifier,TupleExpression(CallExpression(Identifier,Identifier))))))))))
+Query(Statements(PipelineStatement(Pipeline(CallExpression(Identifier,ArgList(Identifier,NestedPipeline(Pipeline(CallExpression(Identifier,ArgList(TupleExpression(CallExpression(Identifier,ArgList(Identifier)))))))))))))
```

Comment:
Potentially one thing we could do is to collapse some of these; e.g. maybe `PipelineStatement` & `Pipeline` are always together and one can be silent. (But no real cost, not high priority)